jetty-client/src/main/java/org/eclipse/jetty/client/HttpRedirector.java

```diff
@@ -85,11 +87,11 @@ public boolean isRedirect(Response response)
     {
         switch (response.getStatus())
```

Comment:
Can't do that, because certain 3XX codes are not really redirects that can be handled here, such as `300 Multiple Choices`, `304 Not Modified`, `305 Use Proxy`, etc.
I can change it into an expression switch though.